WooCommerce/WooCommerceTests/Extensions/WordPressComSiteInfoWooTests.swift

```diff
@@ -4,52 +4,22 @@ import XCTest
 
 final class WordPressComSiteInfoWooTests: XCTestCase {
     func test_is_valid_when_jetpack_is_active_and_connected() {
-        let infoSite = WordPressComSiteInfo(remote: hasJetpack())
-
-        XCTAssertTrue(infoSite.hasValidJetpack)
-    }
-
-    func test_is_not_valid_when_site_does_not_have_jetpack_with_jcp_feature_off() {
-        // Given
-        let featureFlagService = MockFeatureFlagService(isJetpackConnectionPackageSupportOn: false)
-        ServiceLocator.setFeatureFlagService(featureFlagService)
-
         // When
-        let infoSite = WordPressComSiteInfo(remote: doesNotHaveJetpack())
+        let infoSite = WordPressComSiteInfo(remote: hasJetpack())
 
         // Then
-        XCTAssertFalse(infoSite.hasValidJetpack)
+        XCTAssertTrue(infoSite.hasValidJetpack)
     }
 
-    func test_is_valid_when_site_does_not_have_jetpack_with_jcp_feature_on() {
-        // Given
-        let featureFlagService = MockFeatureFlagService(isJetpackConnectionPackageSupportOn: true)
-        ServiceLocator.setFeatureFlagService(featureFlagService)
-
+    func test_is_valid_when_site_does_not_have_jetpack_installed_but_jcp_is_connected() {
         // When
         let infoSite = WordPressComSiteInfo(remote: doesNotHaveJetpack())
 
         // Then
         XCTAssertTrue(infoSite.hasValidJetpack)
     }
 
-    func test_is_not_valid_when_site_has_jetpack_installed_but_not_active_with_jcp_feature_off() {
-        // Given
-        let featureFlagService = MockFeatureFlagService(isJetpackConnectionPackageSupportOn: false)
-        ServiceLocator.setFeatureFlagService(featureFlagService)
-
-        // When
-        let infoSite = WordPressComSiteInfo(remote: hasJetpackInactive())
-
-        // Then
-        XCTAssertFalse(infoSite.hasValidJetpack)
-    }
-
-    func test_is_not_valid_when_site_has_jetpack_installed_but_not_active_with_jcp_feature_on() {
-        // Given
-        let featureFlagService = MockFeatureFlagService(isJetpackConnectionPackageSupportOn: true)
-        ServiceLocator.setFeatureFlagService(featureFlagService)
-
+    func test_is_valid_when_site_has_jetpack_installed_and_inactive_but_jcp_is_connected() {
```

Comment:
thanks for renaming this 👍 the previous name seems incorrect (name says "is not valid" but the assertion is valid)